src/data/astarte_handler.rs

```diff
@@ -299,8 +313,9 @@ mod test {
     use std::collections::HashMap;
     use std::str::FromStr;
 
+    use astarte_device_sdk::error::Error as AstarteSDKError;
```

Comment:
Acronyms in rust have the first letter capitalize, not all of them (eg. Arc, Rc, ...) This could be renamed to SdkError for simplicity.